libs/components/lists/src/lib/modules/repeater/repeater-adapter.service.ts

```diff
@@ -7,31 +7,40 @@ import { SkyRepeaterService } from './repeater.service';
  */
 @Injectable()
 export class SkyRepeaterAdapterService {
-  private get repeaterItemGroupSelector(): string {
-    return '.sky-repeater-item-group-' + this.repeaterService.repeaterGroupId;
+  get #repeaterItemGroupSelector(): string {
+    return '.sky-repeater-item-group-' + this.#repeaterService.repeaterGroupId;
   }
 
-  private host: ElementRef;
+  #host!: ElementRef;
```

Comment:
This is the parent repeater element and is assigned here by the component upon initialization. It should always exist.

Reply:
That may be true in how we're using the service today, but if the consuming component changes how it uses the service, or attempts to call one of the public methods on the service that require the host to be set (before it's set), it would error. It's probably safer to consider it `| undefined` in the service; we can annotate the two `if` checks with `istanbul ignore else`. 👍

Reply:
Fixed